templates/resources/videos-and-webinars.html

```diff
@@ -0,0 +1,143 @@
+<hr class="p-rule">
+<h2 class="p-text--small-caps">Videos and webinars</h2>
+<article class="p-section--shallow">
+  <div class="p-card--resources">
+    <h2>What is MicroK8s?</h2>
+    <div class="icon-container">
+      <a href="https://www.youtube.com/watch?v=v9KI2BAF5QU"><i class="p-icon--video-play is-light"></i></a>
```

Comment:
```suggestion
      <a href="https://www.youtube.com/watch?v=v9KI2BAF5QU"><i class="p-icon--video-play is-light">Watch the What is MicroK8s? video</i></a>
```

Empty links are no accessible. Icon patterns already hide the content allowing linking to contain contextual text.

Reply:
Same for all the media links in this doc